Projects/Arduino/Lcd/lcd.h

```diff
@@ -0,0 +1,42 @@
+/******************************************************************************
+ * @file lcd
+ * @brief driver example a simple lcd
+ * @author mariebidouille
+ * @version 0.0.0
+ ******************************************************************************/
+#ifndef LCD_H
+#define LCD_H
+
+#include "luos.h"
+
+/*******************************************************************************
+ * Definitions
+ ******************************************************************************/
+typedef struct __attribute__((__packed__))
+{
+    union
+    {
+        struct __attribute__((__packed__))
+        {
+            uint8_t mode_display : 1; 
+            uint8_t mode_cursor : 1;
+            uint8_t mode_blink : 1;
+            uint8_t mode_autoscroll : 1;
+            uint8_t mode_scroll_display : 1;
+            uint8_t mode_right_to_left : 1;
+        };
+        uint8_t unmap[2];
```

Comment:
If I understand this structure, you want to use the unmap version as a serialisable version of this lcd_mode_t.
But your config use 6 bit, so you just need an `uint8_t` (witch is the type you choose) but your unmap version use 2 `uint8_t`. You can only use 1 here.